Assets/Scripts/ExpoEventManager.cs

```diff
@@ -141,22 +141,6 @@ public class ExpoEventManager : MonoBehaviourPunCallbacks {
         __mySelectedAvatar = avatar;
     }
 
-    public void DisplayMainMenu() {
-        LeaveEvent();
-        __ResetAvatarPanel();
-    }
-
-    private void __ResetAvatarPanel() {
-        AvatarPanel.SetActive(false);
-        UnselectedAvatarText.SetActive(false);
-        NameIsAvailableText.SetActive(false);
-        DuplicateNameText.SetActive(false);
-        EnterNameText.SetActive(false);
-        NameInputField.Select();

```

Comment:
NameInputField isn't being used anymore, so I think we can remove this public member variable at some point.